test/support.py

```diff
@@ -45,3 +45,10 @@ def setup_make(targetname):
     ispypy = True
 except ImportError:
     ispypy = False
+
+import cppyy
+IS_CLANG_REPL = cppyy.cppexec("""#ifndef __CLING__
```

Comment:
@wlav, we need to detect if we are in clang-repl mode and mark some tests as xfail conditionally. Is that approach okay?